tests/unit/test_benchmark.py

```diff
@@ -42,3 +42,35 @@ def test_progress_bar_updates(tqdm_mock):
 
     # Assert
     tqdm_mock.assert_called_once_with(ANY, total=1, position=0, leave=True)
+
+
+@patch('sdgym.benchmark._score_with_timeout')
```

Comment:
we don't test `_score_with_timeout` at all. Instead of mocking that, can we just mock the `process` calls and make sure the timeout is passed correctly? You don't have to write tests for `_score_with_timeout`, but just test the functionality here.
https://github.com/sdv-dev/SDGym/blob/bd5693c3d0aabf0e862f6532ff51105d220622fd/sdgym/benchmark.py#L286-L288